distribution/docker/gradle/javaAndUnitTests.gradle.kts

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Can we remove this file from the example and reference: https://raw.githubusercontent.com/Knotx/knotx-starter-kit/master/gradle/javaAndUnitTests.gradle.kts

Reply:
✔️ done